pkg/models/action.go

```diff
@@ -0,0 +1,27 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// ActionType represents type of action
+type ActionType string
+
+const (
+	// ActionTypeSUBMIT captures enum value SUBMIT
+	ActionTypeSUBMIT ActionType = "SUBMIT"
+)
+
+// Action is the model for an action on a system intake
+type Action struct {
```

Comment:
Is it okay to have a single action struct for both system intakes and business cases? I think it should be fine, they'll use different endpoints. Related to the question about sharing or separating the db table